tv_apps/browser/js/sync/db.js

```diff
@@ -1394,6 +1396,32 @@ SyncBrowserDB.db = {
   },
 
   /**
+   * Check history by timestamp range at (record.timestamp <= start)
+   * @param {Number} start Start timestamp
+   * @param {Function} callback Runs on complete with an array of history
```

Comment:
I'm confused about the  mismatch of the value passed in the callback and the comment.